### PR TITLE
Issue #176 Update eap-cd addon for EAP-CD-14

### LINKED ISSUE #176: Update eap-cd addon for EAP-CD-14
EAP-CD 14 templates were released two weeks ago (https://github.com/jboss-container-images/jboss-eap-7-openshift-image/tree/eap-cd/templates), the eap-cd addon thus needs an update.

Because the EAP-CD 14 update contains a switch from registry.access.redhat.com to registry.redhat.io, which requires an authentication, this issue is blocked until https://github.com/minishift/minishift/issues/2875 is resolved in Minishift.

### PR DESCRIPTION
**Pull request by agajdosi**
fixes #176

### CODE REVIEW

**Comment by agajdosi** [comment]
retest this please

**Comment by agajdosi** [comment]
retest this please

**Comment by agajdosi** [comment]
Removed branch `pr/158` from this repository and problem with `git fetch` on centos-ci got resolved, waiting now for the results.

**Comment by agajdosi** [comment]
retest this please

**Comment by agajdosi** [comment]
retest this please

**Comment by agajdosi** [comment]
Hi @LalatenduMohanty, @anjannath please take a look on this PR when you have some time.

File: add-ons/eap-cd/eap-cd.addon [hunk 1]
```diff
@@ -1,20 +1,10 @@
...
-oc create -f templates/eap-cd-mysql-s2i.json -n openshift
-oc create -f templates/eap-cd-postgresql-persistent-s2i.json -n openshift
-oc create -f templates/eap-cd-postgresql-s2i.json -n openshift
-oc create -f templates/eap-cd-sso-s2i.json -n openshift
-oc create -f templates/eap-cd-third-party-db-s2i.json -n openshift
-oc create -f templates/eap-cd-tx-recovery-s2i.json -n openshift
+# Templates
+oc apply -f templates -n openshift
+# ImageStream
+oc apply -f imagestream -n openshift
```
Comment (anjannath) [inline add-ons/eap-cd/eap-cd.addon]: :+1:

**Pull request merged**